README.md

```diff
@@ -42,6 +42,7 @@ Any security vulnerabilities can be reported to security@docker.com.
 # Getting started with the Notary CLI
 
 Please get the Notary Client CLI binary from [the official releases page](https://github.com/docker/notary/releases) or you can [build one yourself](#building-notary).
+The version of Notary server and signer should match the Notary CLI to ensure feature compatibility, and all official releases are associated with GitHub tags.
```

Comment:
Should we say that the server version should be >= the client version?  Newer versions of the server should also work with older versions of clients.